tests/autorestart/test_container_autorestart.py

```diff
@@ -41,6 +45,10 @@ def config_reload_after_tests(duthosts, selected_rand_one_per_hwsku_hostname):
     # Config reload should set the auto restart back to state before test started
     for hostname in selected_rand_one_per_hwsku_hostname:
         duthost = duthosts[hostname]
+        # Disable dhcp_server feature for mx topo
+        if tbinfo["topo"]["type"] == "mx":
+            duthost.shell("config feature state dhcp_server disabled")
+            duthost.shell("sudo systemctl restart dhcp_relay.service")
```

Comment:
No need to restart dhcp_relay, after config_reload, it would restart automatically

Reply:
Yes, updated